### DISCUSSION TITLE
SSH key pub key signing failed. Access denied.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#48612] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I've followed the tutrial about generating and configuring the SSH key, but keeps being denied on push request.

I've used the id_rsa.pub for that.

What could be wrong?

**Comment by airtower-luna** [comment]
You'll need to share the exact error message for anyone to start guessing. :wink:

Or even better, run `ssh -vT git@github.com` to get debug info on the SSH connection.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @lukaswsantos ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
